interfaces/cython/cantera/reactor.pyx

```diff
@@ -386,6 +396,13 @@ cdef class ConstPressureReactor(Reactor):
     """
     reactor_type = "ConstPressureReactor"
 
+cdef class ConstPressureMoleReactor(Reactor):
+    """A homogeneous, constant pressure, zero-dimensional reactor with a mole based
+    state vector. The volume of the reactor changes as a function of time in order to
+    keep the pressure constant.
+    """
```

Comment:
I suppose this class should also get the `.. versionadded:: 3.0` annotation. Likewise all of the new `Extensible...Reactor` classes added in this PR.